### ISSUE TITLE
Option to elide package names from output, etc.

### ISSUE THREAD

**Issue by retronym** [jvm-profiling-tools/async-profiler#88]
jfr-flamegraph has a convenient option:

```
             -s, --use-simple-names
              Use simple names instead of qualified names in the stack
              Default: false
```

This reduces the size of dumped CSV files and the resulting SVG files, and makes the flamegraph easier to read (at the cost of losing some accuracy).

One can post process the files with a regex to achieve this, but now that async-profiler supports direct flamegraph output, it would be handy to support it directly.

Java Mission Control has further options to control how stack trace elements are grouped/displayed:

   - Simple vs FQN name for the method owner, as above
   - show/hide method parameters
   - show/hide line number
   - collapse or separate interpreted vs compiled frames.

[perf-map-agent](https://github.com/jvm-profiling-tools/perf-map-agent) displays inlined vs non-inlined frames as distinct (which lets you get a nicer visualization of inlining that `-XX:PrintInlining`)

Oracle Performance Studio lets you group the profiles by line-number, byte-code-index, or assembly PC address.

Where do these features lie on the spectrum of "you're planning to implement it" vs "you'd accept a contribution" vs "out of scope")?

**Comment by apangin** [comment]
"Simple names" option will be rather easy to implement. Even though I've never felt that FQN names make Flame Graphs difficult to read, if you are sure that simple names will help - I can do this.

There is no plan to add Line numbers, BCI or PC for the reasons described in #59. After all, async-profiler is not a microbenchmarking tool.

Distinction between interpreted/compiled/inlined frames is a feature that I *really* want to see in async-profiler. Unfortunately, this is not possible with current implementation based on AsyncGetCallTrace. I will certainly return to this feature after #66.

**Comment by apangin** [comment]
Added `-s`
